ui/l10n/ar.js

```diff
@@ -309,6 +309,7 @@ var dictionary = {
     "label.add.VM.to.tier": "إضافة جهاز إفتراضي في طبقة",
     "label.add.VPN.gateway": "أضف بوابة الشبكة الافتراضية الشخصية",
     "label.add.account": "Add Account",
+    "label.add.accoutordomain": "Add Account/Domain",
```

Comment:
Can you please add the key to other languages as well. See if you can make use of the online translator. Thanks!

Reply:
@nitin-maharana resource strings are added for all the languages